### PR TITLE
updated one key flow

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

**Review by yashovardhan — CHANGES_REQUESTED** [review]
Looks good overall, here are a few things you can look into:

- Check the logout for mfa disabled, update it to setting provider as null.

- Why are we naming the provider as `blockchainProvider`? The terminology is wrong, blockchain providers are infura/ quicknode etc which give you RPC urls. Change it to web3auth provider or just provider.

- For enabling MFA, logging in with web3auth core doesn't guarantee someone will get the MFA enabled screen. Add `MFALevel` as `optional` in `connectTo` function. Also update that in the example and test before adding here.

- A few H2 headings are missing an underline, please check and add.

- You have mentioned you'll be using the web3 library but there are functions for ethers as well, please clearly mention and any of the libraries can be used

**Comment by Saviour1001** [comment]
Fixed the issues.

File: src/pages/guides/one-key-flow.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Using Web3Auth without MFA with future compatibility
+title: Web3Auth One Key Flow without Redirect
 image: "/docs/contents/guides/banners/one-key-flow.png"
 description: Learn how to control web3auth's mfa login user experience.
```
Comment (shahbaz17) [inline src/pages/guides/one-key-flow.mdx:4]: A better description can be added here.

File: src/pages/guides/one-key-flow.mdx [hunk 2]
```diff
@@ -25,6 +25,8 @@ import Tabs from "@theme/Tabs";
 
 ## Introduction
 
+---
+
 Adding Web3Auth Plug and Play SDKs to your application is a great way to authenticate users and reconstruct their private key, and to reconstruct the
```
Comment (shahbaz17) [inline src/pages/guides/one-key-flow.mdx:30]: Starting looks odd to me. Can we align this with our other guides?

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

**Comment by shahbaz17** [comment]
Merging this for now, @Saviour1001 please work on the suggested feedback in a separate PR.

File: src/pages/guides/one-key-flow.mdx (hunk 1 above)
Comment (Saviour1001) [reply src/pages/guides/one-key-flow.mdx:4]: Done

File: src/pages/guides/one-key-flow.mdx (hunk 2 above)
Comment (Saviour1001) [reply src/pages/guides/one-key-flow.mdx:30]: This specific description was added to tell users the flow which was currently and how we are changing it using the one key flow. The context was added after a discussion and thus this is different as compared to other guides

**Pull request merged**